vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -339,7 +347,7 @@ public void performAllStepsForProvidedJsonIfFound(ComparisonRule comparisonRule,
         {
             return Optional.of(Optional.ofNullable(JsonPathUtils.getData(json, jsonPath)));
         }
-        catch (PathNotFoundException e)
+        catch (PathNotFoundException | InvalidJsonException e)
```

Comment:
could you please explain the reason of this change?

Reply:
If we are executing some request and after the last iteration, it is still not valid JSON we should not break the step but fail it;